### ISSUE TITLE
MB - Create feed page with mock posts for first customer milestone

### ISSUE THREAD

**Issue by edonmez01** [bounswe/bounswe2024group8#240]
### Issue Description

For the first customer milestone and presentation, the feed page has to be populated with some mock posts. These posts will utilize the [react-native-3d-model-view](https://github.com/BonnierNews/react-native-3d-model-view) library to display 3D models.

### Issue Due Date

19/10/2024 12:00

### Reviewer

@onurcerli

### Revision Due Date

19/10/2024 12:00

**Comment by edonmez01** [comment]
I completed the initial implementation of the 3D model viewer library. However, because of compatibility issues, I had to eject expo.

I completed the general implementation of the feed page. I dealt with input handling as follows:
- If the user drags their finger along a 3D model presented on the feed page, the 3D model is panned accordingly. No scrolling takes place.
- If the user drages their finger along anywhere on the feed that's not a 3D model, scrolling happens naturally.

However, we still need to find some realistic mock data for the customer presentation. Therefore, I'm not closing the issue yet.

The implementation can be found at the [mobile-post-viewer](https://github.com/bounswe/bounswe2024group8/tree/mobile-post-viewer) branch.
